### PR TITLE
Bare bones

### PR DESCRIPTION
**Pull request by steveklabnik**
This will let people skip ahead if they wish.

### CODE REVIEW

File: src/feel-free-to-skip-ahead.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+chapter. But if you're more excited about writing Rust code than dealing with
+the arcane details of the `x86` platform, don't worry about skipping ahead to
+"[A simple VGA driver]". The chapters up until then will be getting us from
+nothing to a main function written in Rust, and this involves a bunch of
+assembly programming that you may or may not find interesting. In fact, it's so
+common that there's a convention in the hobbyist OS development community to
+have a project called "bare bones" that sets up all of this stuff. So if you're
+not interested in the assembly part, you can skip ahead, and not miss out on
+a whole lot. The chapters up until then will be building up the bare bones, it's
+the chapters after that that dive into more interesting things.
```
Comment (varjmes) [inline src/feel-free-to-skip-ahead.md:20]: I'm not sure "more interesting" things should be used here. Learning `x86` can be super cool and fun (you showed me that) and you're being way too negative about it in this text!

File: src/moving-to-a-bare-bones.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+[dependencies]
+rlibc = "1.0.0"
+x86 = "0.7.0"
+```
+
+Next, run `make`:
+
+```bash
+$ make
+$ make
```
Comment (varjmes) [inline src/moving-to-a-bare-bones.md:48]: Is this a duplicate?

File: src/moving-to-a-bare-bones.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+   Compiling num v0.1.32
+   Compiling serde v0.6.15
+   Compiling serde_json v0.6.1
+   Compiling x86 v0.7.0
+   Compiling intermezzos v0.1.0 (file:///home/steve/src/intermezzOS/kernel/chapter_05)
+ld -n -o target/kernel.bin -T src/asm/linker.ld target/multiboot_header.o target/boot.o target/x86_64-unknown-intermezzos-gnu/release/libintermezzos.a
+```
+
+Whew! Cargo knows not just the packages we're using, but all of the packages
+that those packages are using. Some of the versions might be slightly
```
Comment (varjmes) [inline src/moving-to-a-bare-bones.md:79]: I suggest "Some of the versions might be slightly different to what you see here"

File: src/feel-free-to-skip-ahead.md (hunk 1 above)
Comment (steveklabnik) [inline src/feel-free-to-skip-ahead.md:20]: That's quite fair. I wrote this in kind of a hurry, and am not totally happy with it.

File: src/moving-to-a-bare-bones.md (hunk 2 above)
Comment (steveklabnik) [inline src/moving-to-a-bare-bones.md:48]: yes, copy/paste error

**Comment by steveklabnik** [comment]
this isn't gonna get merged anymore

thanks for the help, @varjmes

**Pull request closed without merging**